modules/config-fields-redis.adoc

```diff
@@ -34,34 +32,40 @@
 |===
 | Field | Type | Description
 | **USER_EVENTS_REDIS** +
-(Required) | Object | Redis connection details for user event handling
-|{nbsp}{nbsp}{nbsp}**.host**  + 
-{nbsp}{nbsp}{nbsp}(Required)| String | The hostname at which Redis is accessible + 
-  {nbsp} + 
+(Required) | Object | Redis connection details for user event handling.
+|**.host**  + 
+(Required)| String | The hostname at which Redis is accessible. + 
 **Example:** +
 `quay-server.example.com` 
-|{nbsp}{nbsp}{nbsp}**.port**  + 
-{nbsp}{nbsp}{nbsp}(Required)| Number | The port at which Redis is accessible + 
-  {nbsp} + 
-**Example:** +
-`6379`
-|{nbsp}{nbsp}{nbsp}**.password** | String | The port at which Redis is accessible + 
-  {nbsp} + 
+|**.password** | String | The password for Redis when using encrypted mode. + 
 **Example:** +
 `strongpassword` 
+|**.port**  + 
+(Required)| Number | The port at which Redis is accessible. + 
+**Example:** +
+`6379`
+| **ssl** | Boolean | Whether to enable TLS communication between Redis and Quay. Defaults to false. 
 |===
 
 
 == Example redis configuration
 
-```
+[source,yaml]
+----
 BUILDLOGS_REDIS:
     host: quay-server.example.com
     password: strongpassword
     port: 6379
+    ssl: true
 
 USER_EVENTS_REDIS:
     host: quay-server.example.com
     password: strongpassword
     port: 6379
-```
\ No newline at end of file
+    ssl: true
+----
+
+[NOTE]
+====
+If `ssl` is set to `false`, you will not need a password for Redis. 
```

Comment:
can't agree on this point, customer can config redis authentication with password even not enabled SS

Reply:
Fixed